layouts/fastly/helix.vcl

```diff
@@ -1758,6 +1758,17 @@ sub hlx_deliver_errors {
      set resp.status = 400;
      set resp.response = "Bad request";
   }
+
+  set resp.http.Vary:X-Debug = "";
```

Comment:
out of curiosity: setting the header to `""`, does it enable the vary on x-debug or disable it?

Reply:
The `*.http.Header-Name:Compontent-Name = "Value";` syntax is used to access cookie values (`Key1=value1,key2=value2`) so setting a compontent without any value will just add the key to the string (or keep it there if already set). Conveniently, this is exactly the syntax the `Vary` header needs, so `set ... = ""` enables it, and `unset ...` disables it.